### ISSUE TITLE
Incorrect script naming in agent_groups script

### ISSUE THREAD

**Issue by vikman90** [wazuh/wazuh-api#27]
```
# agent_groups -h

    /var/ossec/bin/agent_groups [ -l [ -g group_id ] | -c -g group_id | -a (-i agent_id -g groupd_id | -g group_id) [-q] | -s -i agent_id | -r (-g group_id | -i agent_id) [-q] ]

    Usage:
    ./agent_groups.py [-l]                                  # List all groups
    ./agent_groups.py -l -g group_id                        # List agents in group
    ./agent_groups.py -c -g group_id                        # List configuration files in group

    ./agent_groups.py -a -i agent_id -g group_id [-q]       # Set agent group
    ./agent_groups.py -r -i agent_id [-q]                   # Unset agent group
    ./agent_groups.py -s -i agent_id                        # Show group of agent

    ./agent_groups.py -a -g group_id [-q]                   # Create group
    ./agent_groups.py -r -g group_id [-q]                   # Remove group
```

We should remove the `.py` extension.

**Comment by BraulioV** [comment]
Hi @vikman90 and @mgmacias95, 

I think we must change this and use the python module `argparse`, which is specifically made to this purpose.

Best regards,
Braulio.

**Comment by vikman90** [comment]
Temporary fix: https://github.com/wazuh/wazuh/commit/e53462b042cbfa25392a0df1bc9012f220966a48

**Comment by mgmacias95** [comment]
We're keeping this temporary fix for release 3.0. Thanks @vikman90!!
